usaspending_api/etl/es_transaction_template.json

```diff
@@ -1,10 +1,10 @@
 {
-  "index_patterns": ["*"],
+  "index_patterns": ["transactions*"],
   "settings": {
     "index.mapping.ignore_malformed": true,
     "index.max_result_window" : 500000,
     "index": {
-      "number_of_shards" : 5,
+      "number_of_shards" : 3,
```

Comment:
What is the desired impact of reducing the shard count? Would it be beneficial to counter the reduced shards by increasing the replicas?

Reply:
I didn't touch replicas. This speeds up the job. We have too many shards as is. This is still a WIP and dependent on stress testing our baseline shard size - yet to do.

Reply:
Logically it would speed up writes and reindexing. The queries might be negatively impacted

Reply:
@tony-sappe